ec2/connection.py

```diff
@@ -118,3 +120,69 @@ def _parse_config_file(self, path):
                                       format(invalid_options))
 
         return config
+
+
+class ELBConnectionClient(EC2ConnectionClient):
+
+    def client(self):
+        """Represents the ELBConnection Client
+        """
+
+        aws_config_property = (self._get_aws_config_property() or
+                               self._get_aws_config_from_file())
+        if not aws_config_property:
+            return ELBConnection()
+        elif aws_config_property.get('elb_region_name'):
+            region_object = \
```

Comment:
create test in test_connection.py if user provides ec2 region name and elb region name there are no problems.